R/client-cloud.R

```diff
@@ -180,6 +180,10 @@ cloudClient <- function(service, authInfo) {
         json$space <- currentProject$space_id
       }
 
+      if (!is.null(contentCategory)) {
```

Comment:
FWIW I don't think this check is strictly necessary since `contentCategory` is `NULL`, assigning it into a list won't do anything.